src/main/java/seedu/jarvis/ui/MainWindow.java

```diff
@@ -230,6 +244,29 @@ private void setGreeting() {
         greeting.textProperty().bind(logic.getGreeting());
     }
 
+    /**
+     * Sets the week and topic
+     */
+    private void setWeekAndTopic() {
```

Comment:
Awesome way to calculate how many weeks it has been since the start of the semester 👍